### PR TITLE
Update French.properties

### PR DESCRIPTION
**Pull request by Subcher**

### CODE REVIEW

**Comment by yairm210** [comment]
You seem to have a placeholder error - one of the [] you wrote has the wrong text in it

**Comment by HadeanLake** [comment]
```
Placeholder `[cityName]` not found in `French` for entry `Worked by [cityName]`
Placeholder `[numberOfCivs]` not found in `French` for entry `[numberOfCivs] Civilizations in the game`
Placeholder `[numberOfCivs]` not found in `French` for entry `Known and alive ([numberOfCivs])`
Placeholder `[numberOfCivs]` not found in `French` for entry `Known and defeated ([numberOfCivs])`
Placeholder `[stat]` not found in `French` for entry `Cost of purchasing [stat] buildings reduced by [amount]%`
Placeholder `[stat]` not found in `French` for entry `[stats] from all [stat] buildings`
Placeholder `[stat]` not found in `French` for entry `Great Person - [stat]`
Placeholder `[tileImprovement]` not found in `French` for entry `Can construct [tileImprovement]`
Placeholder `[stat]` not found in `French` for entry `+[amount]% Production when constructing [stat] buildings`
```

File: android/assets/jsons/translations/French.properties [hunk 1]
```diff
@@ -739,13 +739,13 @@ Closest city = Ville la plus proche
 Action = Action
 Defeated = Vaincu
  # Requires translation!
-[numberOfCivs] Civilizations in the game = 
+[numberOfCivs] Civilizations in the game = [numberofCivs] Civilisation dans la partie
```
Comment (HadeanLake) [inline android/assets/jsons/translations/French.properties:742]: `[numberOfCivs] Civilizations in the game = [numberOfCivs] Civilisation dans la partie `

File: android/assets/jsons/translations/French.properties [hunk 2]
```diff
@@ -739,13 +739,13 @@ Closest city = Ville la plus proche
...
 Defeated = Vaincu
  # Requires translation!
-[numberOfCivs] Civilizations in the game = 
+[numberOfCivs] Civilizations in the game = [numberofCivs] Civilisation dans la partie 
  # Requires translation!
-Our Civilization: = 
+Our Civilization: = Notre civilisation 
  # Requires translation!
-Known and alive ([numberOfCivs]) = 
+Known and alive ([numberOfCivs]) = Connues et vivantes ([numberofCivs])
```
Comment (HadeanLake) [inline android/assets/jsons/translations/French.properties:746]: `Known and alive ([numberOfCivs]) = Connues et vivantes ([numberOfCivs]) `

case-sensitive

File: android/assets/jsons/translations/French.properties [hunk 3]
```diff
@@ -739,13 +739,13 @@ Closest city = Ville la plus proche
...
+[numberOfCivs] Civilizations in the game = [numberofCivs] Civilisation dans la partie 
  # Requires translation!
-Our Civilization: = 
+Our Civilization: = Notre civilisation 
  # Requires translation!
-Known and alive ([numberOfCivs]) = 
+Known and alive ([numberOfCivs]) = Connues et vivantes ([numberofCivs]) 
  # Requires translation!
-Known and defeated ([numberOfCivs]) = 
+Known and defeated ([numberOfCivs]) = Connues et battues ([numberofCivs])
```
Comment (HadeanLake) [inline android/assets/jsons/translations/French.properties:748]: `Known and defeated ([numberOfCivs]) = Connues et battues ([numberOfCivs]) `

File: android/assets/jsons/translations/French.properties [hunk 4]
```diff
@@ -2703,51 +2703,51 @@ Military Tradition = Tradition militaire
...
 Trade Unions = Unions commerciales
  # Requires translation!
-Maintenance on roads & railroads reduced by 33% = 
+Maintenance on roads & railroads reduced by 33% = Maintenance des routes et des chemins de fer réduits de 33%
 Mercantilism = Mercantilisme
 Entrepreneurship = Entreprenariat
 Patronage = Patronat
  # Requires translation!
-Cost of purchasing [stat] buildings reduced by [amount]% = 
+Cost of purchasing [stat] buildings reduced by [amount]% = Coût d'achat [stats] des bâtiments réduits de [amount]%
```
Comment (HadeanLake) [inline android/assets/jsons/translations/French.properties:2741]: `Cost of purchasing [stat] buildings reduced by [amount]% = Coût d'achat [stat] des bâtiments réduits de [amount]%
`

File: android/assets/jsons/translations/French.properties [hunk 5]
```diff
@@ -2757,26 +2757,26 @@ Sovereignty = Souveraineté
 Scientific Revolution = Révolution scientifique
 Rationalism Complete = Rationalisme complet
  # Requires translation!
-[stats] from all [stat] buildings = 
+[stats] from all [stat] buildings = [stats] de chaque [stats] bâtiment
```
Comment (HadeanLake) [inline android/assets/jsons/translations/French.properties:2760]: `[stats] from all [stat] buildings = [stats] de chaque [stat] bâtiment `
stats stat

File: android/assets/jsons/translations/French.properties [hunk 6]
```diff
@@ -3547,9 +3547,9 @@ Modern Armor = Blindé moderne
 
 Great Artist = Artiste illustre
  # Requires translation!
-Great Person - [stat] = 
+Great Person - [stat] = Personnage illustre
```
Comment (HadeanLake) [inline android/assets/jsons/translations/French.properties:3550]: `[stat]` in translation is required

File: android/assets/jsons/translations/French.properties [hunk 7]
```diff
@@ -3547,9 +3547,9 @@ Modern Armor = Blindé moderne
 
 Great Artist = Artiste illustre
  # Requires translation!
-Great Person - [stat] = 
+Great Person - [stat] = Personnage illustre 
  # Requires translation!
-Can construct [tileImprovement] = 
+Can construct [tileImprovement] = Peut construire
```
Comment (HadeanLake) [inline android/assets/jsons/translations/French.properties:3552]: `[tileImprovement]`  in translation is required

File: android/assets/jsons/translations/French.properties [hunk 8]
```diff
@@ -2703,51 +2703,51 @@ Military Tradition = Tradition militaire
...
  # Requires translation!
-+33% culture in all cities with a world wonder = 
++33% culture in all cities with a world wonder = +33% de culture dans toute les villes dotées d'une Merveille 
 Free Religion = Tolérance religieuse
 Piety Complete = Piété complète
 Classical era = Âge classique
 Piety = Piété
  # Requires translation!
-+[amount]% Production when constructing [stat] buildings = 
++[amount]% Production when constructing [stat] buildings = +[amount]% de production lors de la construction [stats] de bâtiments
```
Comment (HadeanLake) [inline android/assets/jsons/translations/French.properties:2732]: `+[amount]% Production when constructing [stat] buildings = +[amount]% de production lors de la construction [stat] de bâtiments 
`
stat

File: android/assets/jsons/translations/French.properties [hunk 9]
```diff
@@ -642,11 +642,11 @@ Convert production to gold at a rate of 4 to 1 = Convertit la production en or a
 Convert production to science at a rate of [rate] to 1 = Convertit la production en science au taux de [rate] pour 1
 The city will not produce anything. = La cité ne produira plus rien
  # Requires translation!
-Worked by [cityName] = 
+Worked by [cityName] = Employer par [cityName]
```
Comment (HadeanLake) [inline android/assets/jsons/translations/French.properties:645]: What is wrong in this one? Do you have special `i` in your language that looks like normal `i`?

**Comment by yairm210** [comment]
Is this still relevant (you'll fix the problems) or should I close this?

**Comment by yairm210** [comment]
```
com.unciv.testing.TranslationTests > allTranslationsHaveCorrectPlaceholders STANDARD_OUT
    Placeholder `[param]` not found in `French` for entry `+[amount]% Production when constructing [param]`
    Placeholder `[stat]` not found in `French` for entry `+[amount]% Production when constructing [stat] buildings`
    Placeholder `[stat]` not found in `French` for entry `Cost of purchasing [stat] buildings reduced by [amount]%`
    Placeholder `[stat]` not found in `French` for entry `[stats] from all [stat] buildings`
```

**Comment by yairm210** [comment]
Okay after a month on inactivity I'm assuming this is no longer relevant.

**Pull request closed without merging**